embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/payload/Log.kt

```diff
@@ -53,4 +54,14 @@ internal data class Log(
      If span_id is present, trace_id SHOULD be also present. */
     @Json(name = "span_id")
     val spanId: String? = null
-)
+) {
+    internal constructor(logRecordData: LogRecordData) : this(
```

Comment:
Can we add this, given this file will be autogenerated? If there is risk to be overwritten we can move it to LogMapper.

Reply:
I removed it and left an extension function to do the conversion.